rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -884,19 +885,29 @@ object DisplayNode extends Loggable {
 
   private[this] def removeNode(nodeId: NodeId) : JsCmd = {
     val modId = ModificationId(uuidGen.newUuid)
+    import com.normation.rudder.services.servers.DeletionResult._
     removeNodeService.removeNode(nodeId, modId, CurrentUser.getActor) match {
-      case Full(entry) =>
+      case Full(Success) =>
         asyncDeploymentAgent ! AutomaticStartDeployment(modId, CurrentUser.getActor)
         onSuccess
-
+      case Full(PostHookFailed(postHook)) =>
+        logger.error(postHook.msg)
+        onFailure(nodeId, "Node was correctly deleted, but there was an error while running post-deletion hooks", postHook.msg)
+      case Full(PreHookFailed(int : Interrupt)) =>
+        val message = "Node deletion was interrupted because one the pre-deletion hooks exited with interrupt code (100)"
```

Comment:
Hum. 
Currently we have that kind of error: 
```
Exit code=15 for hook: '/opt/rudder/etc/hooks.d/node-pre-deletion/20-warn' with environment variables: [PATH:/home/fanf/java/aspectj/bin:/home/fanf/bin/SublimeText2:/home/fanf/bin:/home/fanf/java/sbt/bin:/home/fanf/java/scala/bin:/home/fanf/java/jdk/bin:/home/fanf/java/ant/bin:/home/fanf/java/mvn/bin:/home/fanf/.gem/ruby/2.2.0/bin:/opt/vagrant/bin:/home/fanf/java/workspaces/rudder-project/rudder-tools/scripts/rudder-dev:/usr/local/bin:/usr/local/sbin:/home/fanf/Applications/.bin:/usr/bin:/home/fanf/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl] [XMODIFIERS:] [GDMSESSION:enlightenment] [LESS_TERMCAP_se:[0m] [DBUS_SESSION_BUS_ADDRESS:unix:path=/run/user/1000/bus] [LESS_TERMCAP_so:[00;47;30m] [XDG_CACHE_HOME:/home/fanf/.cache] [SSH_AGENT_PID:6290] [MOZ_PLUGIN_PATH:/usr/lib/mozilla/plugins] [LC_PAPER:fr_FR.utf8] [PWD:/home/fanf] [E_LIB_DIR:/usr/lib] [PAGER:less] [TERMINOLOGY:1] [E_START:/usr/bin/enlightenment_start] [E_DESKLOCK_UNLOCKED:happened] [GTK_MODULES:canberra-gtk-module] [PANTS:ON] [LC_CTYPE:fr_FR.utf8] [SHLVL:2] [OXYGEN_DISABLE_INNER_SHADOWS_HACK:1] [XTERM_256_COLORS:1] [GREP_COLOR:37;45] [JAVA_HOME:/home/fanf/java/jdk] [TERM:xterm] [XDG_CONFIG_DIRS:/usr/etc/xdg:/etc/xdg] [E_START_TIME:1492519033.1] [XDG_SEAT_PATH:/org/freedesktop/DisplayManager/Seat0] [XDG_SESSION_ID:c2] [E_CONF_PROFILE:standard] [QT_STYLE_OVERRIDE:gtk2] [LC_NAME:fr_FR.utf8] [_:/home/fanf/java/eclipse-neon.2-scala-ide-4.5/eclipse] [DESKTOP:Enlightenment] [XDG_GREETER_DATA_DIR:/var/lib/lightdm-data/fanf] [LIBOVERLAY_SCROLLBAR:0] [DESKTOP_SESSION:enlightenment] [E_SCALE:1,000] [GTK2_RC_FILES:/home/fanf/bin/gtkrc-eclipse] [XDG_MENU_PREFIX:e-] [NLSPATH:/usr/dt/lib/nls/msg/%L/%N.cat] [E_BIN_DIR:/usr/bin] [XAUTHORITY:/home/fanf/.Xauthority] [LC_MEASUREMENT:fr_FR.utf8] [LC_TELEPHONE:fr_FR.utf8] [XDG_DATA_DIRS:/usr/share/enlightenment:/usr/share:/usr/local/share:/usr/share] [E_PREFIX:/usr] [LC_TIME:fr_FR.utf8] [E_ICON_THEME:Adwaita] [LC_MESSAGES:fr_FR.utf8] [MAIL:/var/spool/mail/fanf] [LC_COLLATE:C] [SWT_GTK3:0] [LOGNAME:fanf] [E_DATA_DIR:/usr/share/enlightenment] [SHELL:/bin/zsh] [LC_ADDRESS:fr_FR.utf8] [OLDPWD:/home/fanf] [VISUAL:vim] [TMPDIR:/tmp/fanf] [GDK_SCALE:1] [XDG_SESSION_PATH:/org/freedesktop/DisplayManager/Session0] [_JAVA_OPTIONS:-Dawt.useSystemAAFontSettings=on -Dswing.aatext=true -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel] [LESS:-F -g -i -M -R -S -w -X -z-4] [XDG_SESSION_DESKTOP:enlightenment] [LS_COLORS:rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:] [LC_IDENTIFICATION:fr_FR.utf8] [LESS_TERMCAP_md:[01;31m] [LESS_TERMCAP_me:[0m] [E_RESTART:1] [LESS_TERMCAP_mb:[01;31m] [LC_MONETARY:fr_FR.utf8] [E_TAINTED:NO] [LESS_TERMCAP_us:[01;32m] [XFILESEARCHPATH:/usr/dt/app-defaults/%L/Dt] [LANG:fr_FR.UTF-8] [XDG_SESSION_TYPE:x11] [ASPECTJ_HOME:/home/fanf/java/aspectj] [DISPLAY::0.0] [LESS_TERMCAP_ue:[0m] [E_IPC_SOCKET:/run/user/1000/e-fanf@0/2822] [SCALA_HOME:/home/fanf/java/scala] [E_DESKLOCK_LOCKED:freefreefree] [GDM_LANG:fr_FR.utf8] [XDG_DATA_HOME:/home/fanf/.local/share] [GPG_TTY:/dev/pts/4] [USER:fanf] [XDG_CONFIG_HOME:/home/fanf/.config] [LC_NUMERIC:fr_FR.utf8] [DESK_0_1:1,0] [XDG_SEAT:seat0] [SSH_AUTH_SOCK:/tmp/fanf/ssh-agent.sock] [EDITOR:vim] [DESK_0_0:3,0] [QT_QPA_PLATFORMTHEME:gtk2] [E_LOCALE_DIR:/usr/share/locale] [XDG_RUNTIME_DIR:/run/user/1000] [XDG_VTNR:7] [GREP_COLORS:mt=37;45] [HOME:/home/fanf] [RUDDER_NODEID:00000099-55a2-4b97-8529-5154cbb63a18] [RUDDER_NODE_POLICY_SERVER:root] [RUDDER_NODE_ROLES:] [RUDDER_POLICIES_DIRECTORY_CURRENT:/var/rudder/share/00000099-55a2-4b97-8529-5154cbb63a18/rules] [RUDDER_POLICIES_DIRECTORY_NEW:/var/rudder/share/00000099-55a2-4b97-8529-5154cbb63a18/rules.new] [RUDDER_POLICIES_DIRECTORY_ARCHIVE:/var/rudder/backup/00000099-55a2-4b97-8529-5154cbb63a18/rules].
```
(for pre-hook). 
I'm not sure we want all of that. Having all env var is interesting to spot specific errors (and so are needed in the logs), but here, they can reveal far to much about the internal config of rudder server. 
I believe we should only display:

- the error code (obviously),
- the script name